tests/unit/Core/Content/Media/Thumbnail/ThumbnailServiceTest.php

```diff
@@ -46,14 +47,16 @@ class ThumbnailServiceTest extends TestCase
 
     private Connection&MockObject $connection;
 
+    private ThumbnailSizeCalculator&MockObject $thumbnailSizeCalculator;
+
     protected function setUp(): void
     {
         $this->filesystemPublic = $this->createMock(FilesystemOperator::class);
         $this->filesystemPrivate = $this->createMock(FilesystemOperator::class);
         $this->dispatcher = $this->createMock(EventDispatcherInterface::class);
         $this->indexer = $this->createMock(EntityIndexer::class);
         $this->connection = $this->createMock(Connection::class);
-
+        $this->thumbnailSizeCalculator = $this->createMock(ThumbnailSizeCalculator::class);
```

Comment:
as the calculator is completely stateless i would directly use the implementation here instead of mocking it

Reply:
Done